### PR TITLE
Issue992 movable fins

### LINKED ISSUE #992: movable horizontal fins
The model `IDEAS.Buildings.Components.Shading.HorizontalFins` has a control input that only supports controlling the inclination of the fins. We should add an alternative control input that allows controlling the horizontal displacement of the fins.

### PR DESCRIPTION
**Pull request by Mathadon**
this closes #992 
@icupeiro can you review this?

### CODE REVIEW

File: IDEAS/Buildings/Components/Shading/HorizontalFins.mo [hunk 1]
```diff
@@ -19,32 +22,50 @@ model HorizontalFins "horizontal fins shading"
 
 
 protected
-  Modelica.SIunits.Length dy1 = s-cos(beta_internal)*w-sin(beta_internal)*t;
+  Modelica.SIunits.Length dy1 = s-sin(beta_internal)*w-cos(beta_internal)*t;
```
Comment (Mathadon) [inline IDEAS/Buildings/Components/Shading/HorizontalFins.mo]: @icupeiro I think I spotted a bug here. Can you verify that you agree with this change?

**Comment by Mathadon** [comment]
@icupeiro this pull requests includes #994

File: IDEAS/Buildings/Components/Shading/HorizontalFins.mo (hunk 1 above)
Comment (icupeiro) [reply IDEAS/Buildings/Components/Shading/HorizontalFins.mo]: agreed

**Comment by icupeiro** [comment]
@Mathadon I think this is ready to merge
Implementation of the diffuse heat exchange seems more challenging though. The implementation I proposed time ago seems a bit overcomplex
Any IDEAS on this? :)

**Comment by Mathadon** [comment]
not right now, perhaps something to check literature for

**Pull request merged**